package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "space-shooter",
+  "version": "1.0.0",
+  "description": "![Poster](poster.jpg)",
```

Comment:
What kind of description is this?:)
Description has to be... descriptive:), like "2d rocket shooter game" or something